### ISSUE TITLE
Updating for the latest version of alda

### ISSUE THREAD

**Issue by elydpg** [MadcapJake/language-alda#3]
hello,

as an avid atom user and user of alda, I was delighted when I discovered this package. However, I noticed that it hasn't been updated for awhile, and alda's syntax has changed a lot. Any idea when an update will be rolled out to fix the syntax for the latest versions of alda?

Thanks!

**Comment by MadcapJake** [comment]
[v0.4.0](https://github.com/MadcapJake/language-alda/releases/tag/v0.4.0) should cover everything that's changed. Let me know if I screwed anything up! xD

**Comment by elydpg** [comment]
[Time to update the highlighting again...](https://github.com/alda-lang/alda/releases/tag/1.0.0-rc21)

**Comment by MadcapJake** [comment]
Alright! I'll take a look this afternoon.

**Comment by TetsuHari** [comment]
Is an update to the latest alda syntax coming anytime soon? I'd be really grateful!

**Comment by elydpg** [comment]
Yeah... haven't seen it updated in awhile. Still lots of potential syntax changes before the release, though keeping this up to date would be nice

**Comment by MadcapJake** [comment]
I've been out of the loop wrt Alda and syntax changes but I'll take a look
tonight. If you could summarize syntax changes (or provide me a link to a
summary) that would help greatly!
